jd-geek-source/src/app/services/api.services.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable()
+
+export class ApiService {
+  constructor(private http: HttpClient) {}
+  contentTypeQuery = '?Content-type=%22application%2Fjson%22 HTTP/1.1';
+  headers = new HttpHeaders({'content-type': 'application/json'});
+
+  getTestData() {
+    var format = "JSON";
```

Comment:
Unless you're purposely trying to hoist a variable or specifically need it in the function scope, we should stick to using `let` and `const` as opposed to `var`. I think your intent was to have this in the block scope given the way it's being use, which means you should be using const.